components/script/textinput.rs

```diff
@@ -15,6 +15,12 @@ use std::ops::Range;
 use std::usize;
 use unicode_segmentation::UnicodeSegmentation;
 
+#[derive(PartialEq)]
+pub enum ChangeEditPoint {
```

Comment:
Let's call this `EditPoint`, so we end up with readable names like `EditPoint::Change`.